tcs-service/src/main/resources/db/migration/data/hee/V4.33__update_local_office_names.sql

```diff
@@ -0,0 +1,4 @@
+SET SQL_SAFE_UPDATES = 0;
```

Comment:
Can you describe when this is still needed or remove the variable updates to simplify the script?

Reply:
The same local office data is in Northern Ireland and will need to be updated there.
As this is a data change rather than structural, I'm happy with the scripts sat in the folders for the separate tenants and the NI data to be updated either in a separate PR or in this PR.

Reply:
This is the error I got under safe mode: `Error Code: 1175. You are using safe update mode and you tried to update a table without a WHERE that uses a KEY column.`